lib/src/testing/utils/testrunner.dart

```diff
@@ -746,6 +765,23 @@ void _runTests<PrivateKey, PublicKey>(
       publicKey = pair.publicKey;
       privateKey = pair.privateKey;
     });
+  } else if (c.importKeyException != null) {
+    test('pkcs8 import exception', () async {
+      try {
```

Comment:
formatting of code is weird here. Please use `dart format` you can enable it in vscode with the Dart extension. I think it's even on by default.

Reply:
Tried `dart format`, but nothing seemed to change on this line. A few other parts were formatted though.